2021/08.md

```diff
@@ -99,4 +100,4 @@ Supporting materials includes slides, a link to the proposal repository, a link
 | Dates                    | Location                       |
 |--------------------------|--------------------------------|
 | 2021-10-25 to 2021-10-28 | "London?", GMT                 |
-| 2021-12-14 to 2021-12-15 | Remote: PT                     |
\ No newline at end of file
+| 2021-12-14 to 2021-12-15 | Remote: PT                     |
```

Comment:
Editing the file through the GitHub UI is adding a new line to the end of the file

Reply:
that's fine; it was my error that it lacked one to begin with. all files should always end in trailing newlines.